### DISCUSSION TITLE
Suggestion for Logo

### DISCUSSION THREAD

**Discussion by kassane** [godot-zig/godot-zig#1] [Ideas]
Based on another Godot community, like `godot-rust`, `godot-dlang`, ...

## With Zig logo

![godot-zig](https://github.com/godot-zig/godot-zig/assets/6756180/f21e680b-567d-4b2c-9394-48439d702b90)

or

## Z only

![godot-z](https://github.com/godot-zig/godot-zig/assets/6756180/408bc2ca-96b6-4e1e-bf1f-e5686af27e3c)

### Project (Gimp exported to PSD)
[godot-zig_psd.tar.gz](https://github.com/godot-zig/godot-zig/files/13942641/godot-zig_psd.tar.gz)

cc: @elvisxzhou

**Comment by elvisxzhou** [comment]
Kudos to you!
